pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -114,9 +114,6 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 		return nil, err
 	}
 
-	// ensure services are ordered to simplify generation logic
-	sort.Slice(services, func(i, j int) bool { return services[i].Hostname < services[j].Hostname })
```

Comment:
is this no longer a concern?  Or do you think it should go in the aggregate registry?

Reply:
we sort by time later in function